Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -70,14 +78,25 @@ class SiteTableViewHeader: UITableViewHeaderFooterView {
 class SiteTableViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
     fileprivate let CellIdentifier = "CellIdentifier"
     fileprivate let HeaderIdentifier = "HeaderIdentifier"
-    var profile: Profile! {
-        didSet {
-            reloadData()
-        }
-    }
+    let profile: Profile!
```

Comment:
if profile is set in init. I dont think this needs to be `!`

Reply:
Oops! I am removing the `!` members, fixing....